src/Views/HouseKeepingPanel.vala

```diff
@@ -55,6 +56,19 @@ public class SecurityPrivacy.HouseKeepingPanel : Granite.SimpleSettingsPage {
         };
         download_files_check.add (download_files_grid);
 
+        weak Gtk.IconTheme default_theme = Gtk.IconTheme.get_default ();
+        default_theme.add_resource_path ("/io/elementary/switchboard/plug/security-privacy");
+
+        var screenshot_files_grid = new Gtk.Grid ();
+        screenshot_files_grid.add (new Gtk.Image.from_icon_name ("folder-screenshots-icon", Gtk.IconSize.LARGE_TOOLBAR));
```

Comment:
I hope it's ok to add a new icon here. I did not want to reuse existing folder-pictures, as that might be confusing to the user. I tried to make an icon which keeps the stack of pictures metaphor but shows a miniature desktop instead of the default beach.

Reply:
I think the precedent we have so far is in keyboard settings we gresource the Screenshots app icon, which you can find here: https://github.com/elementary/screenshot/blob/master/data/icons/24.svg

Reply:
Sorry, I did not think to check there! I pulled that icon over here. The final look:

![Screenshot from 2023-06-02 17 34 17](https://github.com/elementary/switchboard-plug-security-privacy/assets/5374391/b99ab9fc-5a45-470b-a0ed-7b88d3eb4fd0)